nodes/src/nodes/tool_bland_ai/bland_driver.py

```
@@ -0,0 +1,229 @@
+                "type": "array",
+                "items": {"type": "string"},
+            },
+        },
+    },
+}
+
+
+class BlandDriver(ToolsBase):
+    """Tool driver that exposes Bland AI voice call capabilities to agents."""
+
+    def __init__(
```

Ruff (0.15.7)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)